packages/zipkin-transport-aws-sqs/README.md

```diff
@@ -0,0 +1,41 @@
+# Zipkin-transport-aws-sqs
+
+![npm](https://img.shields.io/npm/dm/zipkin-transport-aws-sqs.svg)
+
+This is a module that sends Zipkin trace data from zipkin-js to AWS SQS.
+
+## Usage
+
+`npm install zipkin-transport-aws-sqs --save`
+
+```javascript
+const {
+  Tracer,
+  BatchRecorder,
+  jsonEncoder: {JSON_V2}
+} = require('zipkin');
+const {AwsSqsLogger} = require('zipkin-transport-aws-sqs');
+const noop = require('noop-logger');
+
+const AwsSqsRecorder = new BatchRecorder({
+  logger: new AwsSqsLogger({
+    queueUrl: "https://...", //mandatory
+    awsConfig: {
+      credentials: {},
+      accessKeyId: '...',
+      region: '...',
+      secretAccessKey: '...',
+      credentialProvider: {}
+    }, //optional
+    delaySeconds: 0,// optional
+    encoder: JSON_V2, // optional (defaults to JSON_V2)
```

Comment:
I'd remove this parameter and make this to work with JSON_V2 until someone asks for the feature.